src/github.ts

```diff
@@ -0,0 +1,85 @@
+import * as core from '@actions/core';
+import * as github from '@actions/github';
+import { GetResponseDataTypeFromEndpointMethod } from '@octokit/types';
+import { GitHub } from '@actions/github/lib/utils';
+import { extractBumpComment } from './common';
+
+export class Repo {
+  readonly octokit: InstanceType<typeof GitHub>;
+  readonly owner: string;
+  readonly name: string;
+  readonly prNumber?: number;
+
+  constructor() {
+    // Fetch GitHub Action context
+    // from GITHUB_REPOSITORY & GITHUB_EVENT_PATH
+    const { owner, repo, number: prNumber } = github.context.issue;
+    // const { sha, ref } = github.context;
+
+    this.owner = owner;
+    this.name = repo;
+    this.prNumber = prNumber;
+
+    this.octokit = this.getOctokit();
+  }
+
+  isPr(): boolean {
+    return !!this.prNumber;
+  }
+
+  getOctokit(): InstanceType<typeof GitHub> {
+    const ghToken = core.getInput('github-token') || process.env['GITHUB_TOKEN'];
```

Comment:
:+1: 

This `GITHUB_TOKEN` env var is really super good news for us.

Reply:
Yep, cf [this community comment](https://github.community/t/how-to-use-the-github-actions-bot/16535/2).